docs/guide/db-query-builder.md

```diff
@@ -153,6 +153,12 @@ $subQuery = (new Query())->select('id')->from('user')->where('status=1');
 $query->from(['u' => $subQuery]);
 ```
 
+#### Prefixes
+Also a default [[yii\db\Connection::$tablePrefix|tablePrefix]] can be applied. For example if
+ [[yii\db\Connection::$tablePrefix|tablePrefix]] is `tbl_`, `Customer` becomes `tbl_customer` and `OrderItem` becomes `tbl_order_item`.
+ 
+If a table name is given as `{{%TableName}}`, then the percentage character `%` will be replaced with this property value. 
+For example, `{{%post}}` becomes `{{tbl_post}}`.
```

Comment:
This is already covered in `db-dao.md`. http://www.yiiframework.com/doc-2.0/guide-db-dao.html#quoting-table-and-column-names